styles/style.css

```diff
@@ -95,7 +101,62 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
+
+.button-section {
+  display: flex;
+  flex-direction: row;
+}
+
+.button1 {
+  background-color: #c05326;
+  font-size: 1rem;
+  color: white;
+  border-radius: 4px;
+  padding: 5px;
+  height: 30px;
+  width: 120px;
+  border-style: none;
+  margin-right: 4px;
+  
+    
```

Comment:
to make your code more readable its good to be consistent about where you add line breaks